sd/etcd/example_test.go

```diff
@@ -0,0 +1,99 @@
+package etcd
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+	"os"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"golang.org/x/net/context"
+)
+
+// Package sd/etcd provides a wrapper around the coroes/etcd key value store (https://github.com/coreos/etcd) 
+// This example assumes the user has an instance of etcd installed and running locally on port 2379
+func Example() {
+
+	var logger log.Logger
+	{
+		logger = log.NewLogfmtLogger(os.Stdout)
+		logger = log.NewContext(logger).With("ts", log.DefaultTimestampUTC, "caller", log.DefaultCaller)
+	}
```

Comment:
The purpose of a test code is to show users how to wire things up. We don't need to be logging anything. Let's drop this, and all the debug logging below.